pkg/apis/paddlepaddle/v1/trainingjob.go

```diff
@@ -0,0 +1,32 @@
+package v1
```

Comment:
Please follow Go's convention and rename this file from `trainingjob.go` into `training_job.go`.

Reply:
Done